### PR TITLE
[Playvids] Add new extractor

### PR DESCRIPTION
**Pull request by AXDOOMER**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/ytdl-org/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/ytdl-org/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/ytdl-org/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [x] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

This PR adds a new extractor for playvids.com

Many URLs which could trigger edge cases were tested and no bug was found.

### CODE REVIEW

**Comment by dstftw** [comment]
Read coding conventions.

**Comment by AXDOOMER** [comment]
@dstftw I did read them before coding. I also ran flake8 after coding to make sure everything was fine. Can you pinpoint what's wrong?

**Comment by cypheron** [comment]
I guess because of the huge backlog @dstftw doesn't actually check any code, he writes that "read the conventions" nonsense over all PR's without ever mentioning a specific thing. Hope that @dstftw can get off his horse and does some work, otherwise I don't know if my excitement about ytdl being back was really appropriate.

File: docs/supportedsites.md [hunk 1]
```diff
@@ -666,6 +666,7 @@
  - **PlaysTV**
  - **Playtvak**: Playtvak.cz, iDNES.cz and Lidovky.cz
  - **Playvid**
+ - **Playvids**
```
Comment (dirkf) [inline docs/supportedsites.md:669]:
```suggestion
```

Comment (dirkf) [inline docs/supportedsites.md:669]:
```suggestion
```

**Pull request closed without merging**